playbooks/unmount-and-clear-disks.yml

```diff
@@ -0,0 +1,30 @@
+
+---
+- name: unmount and clear all drives
+  hosts: "{{ hosts }}"
+
+  tasks:
+  - name: install dependencies
+    apt: pkg="gdisk"
+
+  - set_fact: disk_list={{ swift.disks }}
+    when: hosts == "swiftnode"
```

Comment:
We are intending to use this playbook on cephnodes as well which is why we went with a more generic approach.